modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/asset/model/DepotAssetRendererFactoryWrapper.java

```diff
@@ -111,6 +114,17 @@ public AssetRenderer<T> getAssetRenderer(long classPK)
 			return assetRenderer;
 		}
 
+		if (assetRendererGroup.isDepot()) {
+			DepotEntry depotEntry = _depotEntryLocalService.getGroupDepotEntry(
+				groupId);
+
+			if (FeatureFlagManagerUtil.isEnabled("LPD-17564") &&
+				(depotEntry.getType() == DepotConstants.TYPE_SPACE)) {
+
+				return assetRenderer;
+			}
+		}
+
```

Comment:
Is this correct? At this point, `assetRendererGroup.isDepot()` will always be true, so the check is redundant? Didn't you mean to say `group.isCMS()` here?